.github/workflows/create-release.yml

```diff
@@ -66,16 +66,23 @@ jobs:
         with:
           name: binaries-dev-proxy-abstractions-${{ github.ref_name }}
           path: ./dev-proxy-abstractions-${{ github.ref_name }}.zip
-      - name: Add install file
-        if: matrix.architecture == 'win-x64' || matrix.architecture == 'win-x86'
-        run: cp ./install.iss ./${{ env.release }}
       - name: Add installer icon
         if: matrix.architecture == 'win-x64' || matrix.architecture == 'win-x86'
         run: cp ./media/icon.ico ./${{ env.release }}
+      - name: Add install file
+        if: (matrix.architecture == 'win-x64' || matrix.architecture == 'win-x86') && !contains(github.ref_name, 'beta')
```

Comment:
Now that I've seen the `contains` check you've added, let's simplify `matrix.architecture == 'win-x64' || matrix.architecture == 'win-x86'` to `contains(matrix.architecture, 'win-')`